.github/workflows/cmp-beta-release-on-label.yml

```diff
@@ -21,8 +21,11 @@ jobs:
     name: Build CMP App
     needs: [run-tests]
     uses: ./.github/workflows/cmp-build-release.yml
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
     permissions:
       checks: write
       pull-requests: write
       contents: write
       id-token: write
+      issues: write
```

Comment:
I notice this has a `permissions: write-all` at line 10 - is that required now we have more fine grained permissions in the jobs themselves?

Reply:
Thanks. I've updated this and tested to check if it affects the beta deploy.